stories/SQFormDialog.stories.tsx

```diff
@@ -45,6 +45,7 @@ const spacing: GridSpacing = 2;
 const defaultArgs = {
   title: 'Default',
   initialValues: {hello: ''},
+  onSave: console.log,
```

Comment:
stray console.log?
Might be useful in the story, if so by all means keep it.

Reply:
Ya, the submit button wasn't doing anything in this story so I just added a console.log